src/index.ts

```diff
@@ -185,13 +204,13 @@ const actionCommon = {
 
     const runnerInfo = `RunnerID:${currentRunnerID}`;
     const runnerLink =
-      `View the [following link](${context.serverUrl}/${owner}/${repo}/actions/runs/${currentRunnerID})` +
+      `View the [following link](https://github.com/${owner}/${repo}/actions/runs/${currentRunnerID})` +
```

Comment:
Isn't this going to break GHES users?

Reply:
Yes, but the problem is we no longer have a `context.serverUrl`, and I didn't immediately find a great way to replace this

Reply:
OK, I found out `process.env.GITHUB_SERVER_URL` holds this information, updated.